code/_onclick/hud/screen_objects.dm

```diff
@@ -223,6 +223,15 @@
 			if(usr.client)
 				usr.client.drop_item()
 
+		if("up hint")
```

Comment:
Ehh. Would be better to make a subtype of `/obj/screen` and put this behavior there.

Reply:
I won't be able to refactor screen objects in time for January update. So I'll just rebase my work on that end to another branch and keep this using the old system.